CppExport/src/visitors/source_visitors/ElementVisitorSource.cpp

```diff
@@ -88,8 +88,9 @@ SourceFragment* ElementVisitorSource::visit(Enumerator* enumerator)
 
 SourceFragment* ElementVisitorSource::visit(MemberInitializer* memberInitializer)
 {
-	notAllowed(memberInitializer);
-	return new TextFragment(memberInitializer);
+	auto fragment = new CompositeFragment(memberInitializer);
+	*fragment << list(memberInitializer->arguments(), ExpressionVisitorSource(data()));
```

Comment:
Erm, a member initializer also has a name, we need to print the whole thing. Use the newer syntax, `{ ..}` instead of `( ... )`